src/status_im/ui/screens/db.cljs

```diff
@@ -241,7 +243,9 @@
                  ::web3-node-version
                  ::webview-bridge
                  ::status-module-initialized?
-                 ::status-node-started?
+                 ::status-node-state
```

Comment:
better use a namespaced keyword as it is clearly a key for the `node` module so `:node/state`